0,org.assertj.core.error.future.ShouldHaveFailed_create_Test:should_create_error_message
1,org.assertj.core.error.future.ShouldHaveFailed_create_Test:should_create_error_message
2,org.assertj.core.error.future.ShouldHaveFailed_create_Test:should_create_error_message
1,org.assertj.core.api.SoftAssertionsTest:should_be_able_to_catch_exceptions_thrown_by_all_proxied_methods
2,org.assertj.core.api.SoftAssertionsTest:should_be_able_to_catch_exceptions_thrown_by_all_proxied_methods
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_was_cancelled
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_was_cancelled
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_is_completed
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_is_completed
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_is_incomplete
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailed_Test:should_fail_if_completable_future_is_incomplete
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_was_cancelled
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_was_cancelled
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_is_completed
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_is_completed
1,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_is_incomplete
2,org.assertj.core.api.future.CompletableFutureAssert_hasFailedWithThrowableThat_Test:should_fail_if_completable_future_is_incomplete
